src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -89,7 +90,9 @@
             "addSelectedQuestions": "Ausgewählte Fragen hinzufügen",
             "status": "Status",
             "quizPointStatistic": "Quiz-Punkte-Statistik",
-            "startTime": "Startzeit",
+            "releaseDate": "Sichtbar ab",
+            "startTime": "Bearbeitungszeit ab",
```

Comment:
You could write `Bearbeitungszeit beginnt ab` if it fits here

Reply:
That was copied directly from [exam.json](https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/i18n/de/exam.json#L275) so maybe it can be better to leave them consistent.